vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -38,9 +38,8 @@ switch -Regex ($windowsSku) {
             "mcr.microsoft.com/windows/servercore:ltsc2022",
             "mcr.microsoft.com/windows/nanoserver:ltsc2022",
 
-            # NPM (Network Policy Manager) & CNS (Container Networking Service) Owner: jaer-tsun (Jaeryn)
-            "mcr.microsoft.com/containernetworking/azure-npm:v1.4.34",
-            "mcr.microsoft.com/containernetworking/azure-cns:v1.4.35"
```

Comment:
this was here by accident before?

Reply:
yes CNS gets added to both 2019 and 2022 below so no need to add it to 2022 explicitly and then add it again later (no-op but opportunity for mistakes later)